addons/binding/org.openhab.binding.russound/src/main/java/org/openhab/binding/russound/internal/rio/AbstractBridgeHandler.java

```diff
@@ -77,6 +98,38 @@ public SocketSession getSocketSession() {
     }
 
     /**
+     * Returns the {@link RioPresetsProtocol} for this {@link Bridge}. The default implementation is to look in the
+     * parent
+     * {@link #getPresetsProtocol()} for the {@link RioPresetsProtocol}
+     *
+     * @return a {@link RioPresetsProtocol} or null if none exists
+     */
+    @SuppressWarnings("rawtypes")
```

Comment:
Forgot to reply on this one - I'm walking up the tree of AbstractBridgeHandlers, won't have access to the 'type' they are nor do I care about the 'type'.  I could make AbstractBridgeHandler extend some other (untyped) abstract base that implements the 'untyped' function - but it's not worth the time/code/effort for this one function.